components/coins-status/index.js

```diff
@@ -8,70 +9,33 @@ const Coins = (props) => {
     changeCoins(props.coin.value);
   }, [props.coin.value, changeCoins]);
   return (
-    <div className="cointype-indicator">
-      <div className="tooltip">
-        <div className="coin rotate-vert-center ">
+    <div className={styles.cointypeIndicator}>
+      <div className={styles.tooltip}>
+        <div className={`coin ${styles.rotateVertCenter}`}>
           <p> {coins} </p>
         </div>
-        <span className="tooltiptext">{props.coin.name}</span>
+        <span className={styles.tooltiptext}>{props.coin.name}</span>
       </div>
-
-      <style jsx>
-        {`
-          .cointype-indicator {
-            display: flex;
-          }
-          .coin {
-            background-color: ${props.coin.color};
-            width: 5rem;
-            height: 5em;
-            border-radius: 50%;
-            box-shadow: 3px 3px 3px #000;
-            border: 1px solid ${props.coin.borderColor};
-            position: absolute;
-          }
-
-          .coin > p {
-            padding-top: 0.5rem;
-            text-shadow: -1px -1px 1px #fff, 1px 1px 1px #000;
-            color: #9c8468;
-            font-weight: bold;
-            font-size: 1.5rem;
-            opacity: 0.7;
-            transform: translate(20%, -10%);
-          }
```

Comment:
Why are some of the things coming from module.css and some of them from styled components?

Reply:
We are using both of them in our project so for ease at writing styles styled component is something that I prefer but they have limitations with media queries and that requires changing config and adding plugins but I didn't want to add extra dependency.